std/functional.d

```diff
@@ -26,6 +26,9 @@ $(TR $(TH Function Name) $(TH Description)
     $(TR $(TD $(D $(LREF lessThan)), $(D $(LREF greaterThan)), $(D $(LREF equalTo)))
         $(TD Ready-made predicate functions to compare two values.
     ))
+    $(TR $(TD $(D $(LREF lessOrEqualThan)), $(D $(LREF greaterOrEqualThan)), $(D $(LREF notEqualTo)))
```

Comment:
@burner Does your browser not wrap the column?

JackStouffer is right, it looks like a lazy copy-paste job right now. There are better solutions for this problem.